### DISCUSSION TITLE
ActiveMq Classic

### DISCUSSION THREAD

**Discussion by NaceHudobivnik** [kedacore/keda-docs#639] [Q&A] [answered]
Hello.

I've gone through the list of scalers for KEDA and realized that there is one for ActiveMQ Artemis, but not for ActiveMQ Classic.
They are two flavors of the ActiveMQ and I would need to have support for the ActiveMQ Classic.

I would assume we'd need to implement a new scaler, probably taking the basis from the Artemis one?

**Comment by zroubalik** [comment, marked as answer]
Hi,

you should thank @melisatanrverdi :) She implemented this scaler for the next release:
https://github.com/kedacore/keda/pull/2305
